src/test/webview/ui/playerComponent.test.ts

```diff
@@ -52,10 +62,44 @@ describe('player', () => {
     });
 
     test('change volume when volume-bar is changed', () => {
-        const volumeBar = <HTMLInputElement>document.getElementById("volume-bar");
+        let volumeBar = <HTMLInputElement>document.getElementById("volume-bar");
+
+        volumeBar.value = "0";
+        volumeBar.dispatchEvent(new Event('input'));
+        expect(playerService.volume).toBe(1.0);
+        volumeBar.value = "-20";
+        volumeBar.dispatchEvent(new Event('input'));
+        expect(playerService.volume).toBe(0.1);
+        volumeBar.value = "-80";
+        volumeBar.dispatchEvent(new Event('input'));
+        expect(playerService.volume).toBe(0.0);
+
+        playerComponent.dispose();
+        playerService.dispose();
+
+        // create playerComponent again to change setting
```

Comment:
I agree with splitting the test, but I am encountering difficulties in writing this test.
This is not directly related with splitting the test or not, though.

In the beforeAll() function, I create a defaultSettings instance in the same way as in analyzeSettingsService.test.ts.
At that time, volumeUnitDb is set to either true or false.
Thus, I need to dispose of the instance and create it again when testing the setting opposite to the default.
I don't believe this is best practice.

Which option do you think is most appropriate?

1. Proceed as above.
2. Avoid using beforeAll() and create a defaultSettings instance at the beginning of each test.
3. Split the test file based on whether volumeUnitDb is true or false.

Reply:
I see, it's difficult to write tests because my Service code is dirty.
Because it listens to window events and searches for its own child elements using document.getElementById, it is not designed to allow multiple playerServices to exist.
Because of this, You can't choose to simply create another PlayerService instance.
I'll fix this separately.

It may be a good idea to use beforeEach for this PR.

Reply:
Thank you for your advice.

This time, I tried to make modifications as [latest commit](https://github.com/sukumo28/vscode-audio-preview/pull/55/commits/36e7809afa3da94adf3adc7c985ba9625c2c6772#diff-a726bb2dcf6369161dfedc5556dc9f45a78721d8e72e99cd0abf39eb514ec705L8).
I split the test for the player into two "describe" sections. 
One section is for general test with "volumeUnitDb = false", and the other is additional test for "volumeUnitDb = true".
By doing this, I can use the different instances for the volume-bar test.
Note that although it may appear that the test code has been completely modified, the former section is just an indentation of the original code.

 Are these modifications good?

Reply:
> Are these modifications good?
Yes. Thank you for fixing.

Because the original code was dirty, the implementation ended up being a bit exaggerated compared to what we wanted to do.
This implementation is fine because it's not your PR's fault.
Can I merge this? Or is there still something else you want to push?

Reply:
Thank you for your confirmation.

> Or is there still something else you want to push?

Nothing else for this PR. All done!